applications/ContactStructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -50,7 +50,6 @@
 // Builders and solvers
 #include "solving_strategies/builder_and_solvers/builder_and_solver.h"
 #include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h"
-#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_constraints.h"
```

Comment:
Please replace it with the new one instead of removing it

Reply:
ok .. doing it now .. !